configs/components/razor-server.rb

```diff
@@ -75,16 +74,11 @@
   if settings[:pe_package]
     case platform.servicetype
     when "systemd"
-      # Add JAVA to razor-server.env so it can find pe-java correctly
-      install_commands.push("sed -i '/^LANG=en_US.UTF-8$$/ a JAVA=#{settings[:server_bindir]}/java' #{settings[:prefix]}/razor-server.env")
```

Comment:
I added JAVA to the sysconfig template in the puppetlabs-pe_razor module. Is that enough? Or should the package have that as well?

Reply:
I think the package should have it too, unless the package (on upgrades) will overwrite what the module lays down?